docs/building-a-form/creating-a-form-config-file.md

```diff
@@ -4,11 +4,63 @@ Your form is generated from a JSON Schema configuration file called `form.js`, a
 
 ### In this guide
 
+- [Building a simple form](#building a simple form)
 - [Example `form.js` file](#example-form.js-file)
 - [About the `schema` and `uiSchema` objects](#about-the-schema-and-uischema-objects)
 - [Configuring `uiSchema` using rjsf options](#configuring-uischema-using-rjsf-options)
 - [Configuring `uiSchema` using US Form System options](#configuring-uischema-using-us-form-system-options)
 
+### Building a simple form
+
+The US Forms System contains a `FormApp` component that you configure and control via a `formConfig` object. In the starter app, the file that exports this object is located in `/src/js/config/form.js` and included in `/src/components/Form.jsx`. where `FormApp`is used. When building an app from scratch you can define and include these components where it's convenient.
```

Comment:
I might rephrase the opening paragraph here. I feel like the first concepts someone should understand before building a form is that they describe the form in `formConfig` and they need to render the top-level React component, `FormApp`, somewhere. I might write:

"In order to build a form, you'll use the configuration file to specify everything your form needs: questions, how questions are grouped together, content on the page, data validation, etc. In [the starter app](https://github.com/usds/us-forms-system-starter-app), the file that exports this configuration object is located at `/src/js/config/form.js`.

Most of the time you'll just be working within the config file to describe the form you need and you won't be writing React components. However, there are 2 cases where you'll need to work with React components directly:
1. **Rendering the top-level React component**: US Forms System uses a top-level React component, `FormApp`, to pass the `formConfig` object, exported from your config file, down to the rest of the components that are rendered. In order for the React component hierarchy to properly render, you'll need to include `FormApp` in _your_ app's top-level component, and pass it the `formConfig` object as a prop. See [how the starter app does this](https://github.com/usds/us-forms-system-starter-app/blob/master/js/components/Form.jsx) for an example.
2. **Creating custom components**: There may be times where you'll want to create your own custom React components to include in the form config. Examples may include creating a custom page you want to render (e.g., an introduction page), creating a component for content (e.g., a warning message) you want to render within a page, or creating a new component to override one of our default ones. In these cases you'll be building React components from scratch."

Something like that?